code/modules/client/preference/preferences.dm

```diff
@@ -138,50 +138,9 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	//Disabilities
 	var/disabilities= 0			//Stores bits for DISABILITY_FLAGs
 	var/disabilities_cures = 0	//Stores bits for CURE_FLAGs
-	var/list/dname_list = list("Nearsighted", "Colourblind", "Blind", "Deaf", "Mute", "Obese", "Swedish Accent", "Chav Accent", "Lisp", "Dizziness")
-	var/list/dflag_list = list(DISABILITY_FLAG_NEARSIGHTED, DISABILITY_FLAG_COLOURBLIND, DISABILITY_FLAG_BLIND, DISABILITY_FLAG_DEAF, DISABILITY_FLAG_MUTE, DISABILITY_FLAG_FAT, DISABILITY_FLAG_SWEDISH, DISABILITY_FLAG_CHAV, DISABILITY_FLAG_LISP, DISABILITY_FLAG_DIZZY)
-	var/list/dcure_list = list(CURE_FLAG_NEARSIGHTED, CURE_FLAG_COLOURBLIND, CURE_FLAG_BLIND, CURE_FLAG_DEAF, CURE_FLAG_MUTE, CURE_FLAG_FAT, CURE_FLAG_SWEDISH, CURE_FLAG_CHAV, CURE_FLAG_LISP, CURE_FLAG_DIZZY)
-	var/list/dblock_list = list()
-
-
-	var/datum/dna/gene/disability/d_gene
-	character.dna.SetDNAState(GLOB.fatblock, TRUE, DNA_SE, TRUE)
-
-
-GLOB.glassesblock
-
```

Comment:
I didn't know I committed this in the first place. I had put this here for reference for myself x__x;